providers/fips/fipsprov.c

```diff
@@ -23,6 +23,7 @@
 #include "prov/seeding.h"
 #include "self_test.h"
 #include "internal/core.h"
+#include "e_os.h"
```

Comment:
"Oh look what the cat dragged in..." 😄

Reply:
Personally I don't like it (and probably it's worth considering a separate small header for locale-related defines in master) but making a duplication in 3.0 looks like a bad choice.

Reply:
> > "Oh look what the cat dragged in..." smile
> 
> Is that a good thing or a bad thing?

Some of us consider that header file an ugly duckling...  it drags in quite a lot of cruft.  But we don't have a sane alternative for now

Reply:
:disappointed:

Reply:
> Some of us consider that header file an ugly duckling... it drags in quite a lot of cruft. But we don't have a sane alternative for now

IMO the solution for e_os.h would be to clean it up but not to remove it altogether.